st2common/st2common/exceptions/db.py

```diff
@@ -20,6 +20,10 @@ class StackStormDBObjectNotFoundError(StackStormBaseException):
     pass
 
 
+class StackStormDBObjectFoundError(StackStormBaseException):
```

Comment:
The name doesn't really suggest it should an error. Maybe something like StackStormDBObjectAlreadyExistsError?